about/index.html

```diff
@@ -115,7 +115,7 @@
 							<li><a target="_self" href="../#subscribe">Subscribe</a></li>
 							<li><a target="_self" href="https://blog.fossasia.org">Blog</a></li>
 							<li class="has-dropdown"><a href="https://events.fossasia.org">Events</a>
-								<ul class="nav-dropdown" style="min-height: 575px">
+								<ul class="nav-dropdown" style="min-height: 575250pxpx">
```

Comment:
@STEdirisuriya  Why are there two different values??

Reply:
@sk9331657 I am really sorry. It's a small typo. I fixed it now.